src/containers/Login.jsx

```diff
@@ -41,70 +44,83 @@ class Login extends Component {
       history: { push },
     } = this.props;
     return (
-      <div className="container">
-        <div className="subContainer">
-          <Scenic />
-        </div>
-        <div className="subContainer">
-          <div className="containerPadding">
-            <h1>
-              <span className="sh">
-                SLO Hacks
-              </span>
-              <br />
-              Attendee Application
-            </h1>
-            <form onSubmit={handleSubmit(this.onSubmit.bind(this))}>
-              <Field
-                label="Email"
-                name="email"
-                type="text"
-                placeholder="Email"
-                component={TextInput}
-              />
-              <Field
-                label="Enter a password (8 or more characters)"
-                name="password"
-                type="password"
-                placeholder="Password"
-                component={TextInput}
-              />
-              {errorMessage ? (
-                <FormHelperText style={{ marginBottom: '1rem' }} error>
-                  {errorMessage}
-                </FormHelperText>
-              ) : null}
-              {loading ? <CircularProgress color="primary" /> : (
+      <PageContainer>
+        <SectionHeaderContainer>
+          <SectionHeader>Welcome to the SLO Hacks 2020 application system!</SectionHeader>
+          <SectionText>
+            {`
+             If you're applying to attend SLO Hacks 2020, go ahead and 
```

Comment:
Is this part supposed to show up? I see that this login component is being linked from the router but none of these section headers/texts are showing up.

Reply:
Did you check out my branch/PR, I'm not sure what's going on.

Reply:
lmao i was on the right repo but wrong branch, my bad. lemme re-review